### PR TITLE
Add check if the systemtest was a success.

### LINKED ISSUE #164: Artifact Jobs have Successfully Triggered
### Description
As a system test designer I want to check that artifact jobs have successfully triggered and then remove the subscriptions and jobs so that the system test environment is cleaned after the system tests. 

- [ ] Loop for some time to see artifact jobs successfully triggered.
- [ ] Remove subscriptions and jobs.

Part of #162

### PR DESCRIPTION
**Pull request by e-edling-ericsson**
### Applicable Issues

closes #164 

### Description of the Change

Add a check that loops and checks if all jenkins jobs has been triggered.

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: 
Erik Edling erik.edling@ericsson.com

### CODE REVIEW

File: src/systemtest/java/com/ericsson/ei/systemtest/utils/StepsUtils.java [hunk 1]
```diff
@@ -169,6 +174,53 @@ public static void deleteSubscriptions(String frontendBaseUrl, String backendBas
...
+     * @param jenkinsJob
+     * @throws Exception
+     */
+    public static void hasJenkinsJobBeenTriggered(String jenkinsJob) throws Exception {
+        long maxTime = System.currentTimeMillis() + 60000;
+
+        while(System.currentTimeMillis() < maxTime) {
+            try {
+                JSONObject status = jenkinsManager.getJenkinsBuildStatusData(jenkinsJob);
+                LOGGER.info(jenkinsJob + " was triggered.");
```
Comment (tobiasake) [inline src/systemtest/java/com/ericsson/ei/systemtest/utils/StepsUtils.java]: Doesn't status include some relevant status info that can be valuable for the logs.

File: src/systemtest/java/com/ericsson/ei/systemtest/artifactflow/ArtifactFlowSteps.java [hunk 2]
```diff
@@ -89,18 +91,22 @@ public void we_send_the_to_eiffel_intelligence_for_creation(String subscriptionN
...
-    @Then("^everything should be fine\\.$")
-    public void everything_should_be_fine() {
+    @When("^all jenkins jobs has been triggered$")
+    public void the_jenkins_job_has_been_triggered() throws Throwable {
+        StepsUtils.hasJenkinsJobsBeenTriggered(jenkinsJobNames);
+    }
 
+    @Then("^the test was a succcess$")
+    public void the_test_was_a_succcess() {
+        //Just for cucumber to make sence
```
Comment (tobiasake) [inline src/systemtest/java/com/ericsson/ei/systemtest/artifactflow/ArtifactFlowSteps.java]: We have an open discussion about this in a different Pull-request.

Comment (e-edling-ericsson) [reply src/systemtest/java/com/ericsson/ei/systemtest/artifactflow/ArtifactFlowSteps.java]: Ye, i'll wait until we have decided :)

File: src/systemtest/java/com/ericsson/ei/systemtest/artifactflow/ArtifactFlowSteps.java [hunk 3]
```diff
@@ -38,6 +41,13 @@ public void configurations_are_provided() {
 
     @Given("^a jenkins job '\\\"([^\\\"]*)\\\"' from '\"([^\"]*)\"' is created$")
     public void a_jenkins_job_from_is_created(String jenkinsJobName, String scriptFileName) throws Throwable {
+        /*JenkinsXmlData jenkinsXmlData = new JenkinsXmlData()
+                .addGrovyScript("")
+                .addBuildParameter("EVENT_ID")
+                .addBuildParameter("TEST_CASE_NAME");
+        
+        String test = jenkinsXmlData.getXmlAsString();*/
+        System.out.println();
```
Comment (e-edling-ericsson) [inline src/systemtest/java/com/ericsson/ei/systemtest/artifactflow/ArtifactFlowSteps.java]: When @a-breid-ericsson is done with his PR in eiffel-commons, i will change the out commented lines. I won't merge like this :) Don't mind this part

Comment (a-breid-ericsson) [reply src/systemtest/java/com/ericsson/ei/systemtest/artifactflow/ArtifactFlowSteps.java]: https://github.com/eiffel-community/eiffel-commons/pull/14

**Comment by e-edling-ericsson** [comment]
In the latest commit i also made it possible to tell the jobs what parameters they should have(if any)

**Pull request merged**